model/common/src/icon4py/model/common/interpolation/interpolation_factory.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         self._config = {
             "divavg_cntrwgt": 0.5,
             "weighting_factor": 0.0,
-            "nudge_max_coeffs": 0.375,
+            "nudge_max_coeffs": 0.375,  # TODO why the extra `s` at the end? is this a typo?
```

Comment:
I would say so. It is called exactly like this in the function that uses it in `compute_nudgecoeffs.py` (which computes a field...)

Reply:
you agree it's a typo?

Reply:
I renamed it